index.html

```diff
@@ -1,446 +1,291 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./css/main.css" />
-    <link rel="icon" href="./favicon.ico" />
-    <title>Aziz Mejri - Portfolio</title>
-  </head>
-  <body>
-    <header>
-      <h1 id="logo">Aziz Mejri</h1>
-      <button type="button" class="menu-bars">
-        <i class="fas fa-bars"></i>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./css/main.css" />
+  <link rel="icon" href="./favicon.ico" />
+  <title>Aziz Mejri - Portfolio</title>
+</head>
+
+<body>
+  <header>
+    <h1 id="logo">Aziz Mejri</h1>
+    <div id="hamburger-menu" class="hamburger">
+      <button id="hamburger" type="button">
+        <span class="ham-top"></span>
+        <span class="ham-mid"></span>
+        <span class="ham-bottom"></span>
       </button>
```

Comment:
Great job implementing the mobile menu 👍 However, I noticed that while implementing the **cross** button as suggested by previous reviewer, the style of the hamburger icon changed a bit. Let's try to understand this with pictures mentioned below 🔽

## Your design now
![image](https://user-images.githubusercontent.com/106119105/211099769-8dbbeabf-c90d-476a-b3ba-1a001bba5498.png)

## Your design before
![image](https://user-images.githubusercontent.com/106119105/211099903-53e4e353-7d8d-49ec-8765-e6c7ed466fa8.png)

- [ ] Don't worry, that is not a big deal 😃. The good news is that you can easily fix this with a little bit of styling. For example, you can try adding something like below to your CSS file to fix this 💪

```
#hamburger {
  background: transparent;
  border: none;
};
```